docs/mast/mast_obsquery.rst

```diff
@@ -456,10 +459,33 @@ Getting a list of S3 URIs:
    ...                                         productSubGroupDescription='DRZ')
    >>> s3_uris = Observations.get_cloud_uris(filtered)
    >>> print(s3_uris)
-   ['s3://stpubdata/hst/public/jbev/jbeveo010/jbeveo010_drz.fits', 's3://stpubdata/hst/public/jbev/jbeveo010/jbeveo010_drz.fits', 's3://stpubdata/hst/public/jbev/jbevet010/jbevet010_drz.fits', 's3://stpubdata/hst/public/jbev/jbevet010/jbevet010_drz.fits']
+   ['s3://stpubdata/hst/public/jbev/jbeveo010/jbeveo010_drz.fits', 's3://stpubdata/hst/public/jbev/jbevet010/jbevet010_drz.fits']
    ...
    >>> Observations.disable_cloud_dataset()
 
+Alternatively, this workflow can be streamlined by providing the query criteria directly to `~astroquery.mast.ObservationsClass.get_cloud_uris`.
+This approach is recommended for code brevity. Query criteria are supplied as keyword arguments, and filters are supplied through the 
+``filter_products`` parameter. If both ``data_products`` and query criteria are provided, ``data_products`` takes precedence.
+
+.. doctest-skip::
```

Comment:
Why skipping this example

Reply:
Good point, this shouldn't be skipped. New commit reflects this!